src/util/util.h

```diff
@@ -83,6 +83,16 @@
         {"gid", 0, POPT_ARG_INT, &gid, 0, \
           _("The group ID to run the server as"), NULL},
 
+extern int socket_activated;
+extern int dbus_activated;
+
+#define SSSD_RESPONDER_OPTS \
+        {"socket-activated", 0, POPT_ARG_NONE, &socket_activated, 0, \
+          _("Informs that the responder has been socket-activated"), NULL }, \
+        {"dbus-activated", 0, POPT_ARG_NONE, &dbus_activated, 0, \
+          _("Informs that the responder has been dbus-activated"), NULL },
+
+
```

Comment:
It would just add more ifdefs in our code for no good reason. 
I really would prefer to have it as it is.

And the extra line will be fixed in the next patch set.